### PR TITLE
Advise users to increase OS opened files limit

### PR DESCRIPTION
**Pull request by bostko**

### CODE REVIEW

File: guide/ops/troubleshooting/increase-system-resource-limits.md [hunk 1]
```diff
@@ -4,6 +4,16 @@ title: Increase System Resource Limits
 toc: /guide/toc.json
 ---
 
+If you encounter the following error:
+
+    Caused by: java.io.IOException: Too many open files
+            at java.io.UnixFileSystem.createFileExclusively(Native Method)[:1.8.0
+
+Please check limit for opened files `cat /proc/sys/fs/file-max` and increase it.
```
Comment (drigodwin) [inline guide/ops/troubleshooting/increase-system-resource-limits.md]: Please check the limit for open files in `/proc/sys/fs/file-max`

File: guide/ops/troubleshooting/increase-system-resource-limits.md [hunk 2]
```diff
@@ -4,6 +4,16 @@ title: Increase System Resource Limits
...
+You can increase limit for maximum opened files by setting in `/etc/sysctl.conf` `fs.file-max = 700000`
```
Comment (drigodwin) [inline guide/ops/troubleshooting/increase-system-resource-limits.md]: You can increase the maximum limit of opened files by setting `fs.file-max` in `/etc/sysctl.conf`

File: guide/ops/troubleshooting/increase-system-resource-limits.md [hunk 3]
```diff
@@ -4,6 +4,16 @@ title: Increase System Resource Limits
...
+`sudo sysctl -p` to apply the changes.
```
Comment (drigodwin) [inline guide/ops/troubleshooting/increase-system-resource-limits.md]: and then running `sudo sysctl -p` to apply the changes.

**Comment by drigodwin** [comment]
This is a good addition, I've suggested some minor rewording. Thanks @bostko!

File: guide/ops/troubleshooting/increase-system-resource-limits.md (hunk 1 above)
Comment (bostko) [inline guide/ops/troubleshooting/increase-system-resource-limits.md]: @drigodwin do you insist on this rewording?

**Comment by bostko** [comment]
@drigodwin  thank you for the suggestions!
I am hesitant to reword the first sentence WDYT?

File: guide/ops/troubleshooting/increase-system-resource-limits.md (hunk 1 above)
Comment (drigodwin) [inline guide/ops/troubleshooting/increase-system-resource-limits.md]: Well the missing `the` in `Please check limit` is definately required for proper english, if you want to keep `cat` in there, that's fine.

**Pull request merged**